danger/Dangerfile

```diff
@@ -0,0 +1,16 @@
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = github.pr_title.include? "#trivial"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vividmuimui/danger-sample/blob/1ecef35c62a1da4354f6e976af402db5b5bcf679/danger/Dangerfile#L3"></span>Lint/UselessAssignment: Useless assignment to variable - <code>declared_trivial</code>.</td>
    </tr>
  </tbody>
</table>